app/frontend/src/features/auth/signup/CompleteSignupForm.tsx

```diff
@@ -318,18 +318,17 @@ export default function CompleteSignupForm() {
               )}
             />
             <div>
-              <TOS />
-              <Button
-                classes={{
-                  label: authClasses.buttonText,
-                  root: authClasses.button,
-                }}
-                loading={loading}
-                onClick={() => setAcceptedTOS(true)}
-                disabled={acceptedTOS}
-              >
-                {acceptedTOS ? THANKS : ACCEPT}
-              </Button>
+              <TOSLink />
```

Comment:
It would be good to put this in a sentence:

"To continue, please read and accept the [Terms of Service]."

Or even just put it as part of the checkbox "I accept the [Terms of Service]"?

Reply:
Uhh this is a bit of a pain in the arse, as I want the link to be reused across the few places but putting it into a sentence makes as part of the checkbox label makes extracting the text into constans awkward, so think I'll stick with the first even though it's a bit of repetition

Reply:
Sounds good